src/raftstore/store/msg.rs

```diff
@@ -59,6 +63,8 @@ pub enum Msg {
 
     ReportUnreachable { region_id: u64, to_peer_id: u64 },
 
+    ReportStorageGc { region_id: u64, peer_id: u64 },
```

Comment:
Oh, my fault, this is for GC, not for region merge. 
I think we should merge the master.